src/igvfd/schemas/changelogs/biosample.md

```diff
@@ -4,3 +4,4 @@
 
 * Replaced related_biosamples and related_biosamples_relationship with pooled_from and part_of
 * organism property renamed to taxa
+* Bumped date_obtained property from biosample to sample.
```

Comment:
I think we should use "Moved" instead of "Bumped"

Reply:
so probably something like: "*date_obtained* property definition was moved from the Biosample schema to Sample schema. It is introduced in Biosample through the mixin mechanism"